1.0/warn-objects/warn03_zero_padded_versions/v0002/inventory.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "bb123cd4567",
-  "head": "v0002",
```

Comment:
v0002 is correct; this is the inventory as it was when v0002 was the most recent version.

Reply:
Yup, my bad. I create old version inventories by editing the last one down and forget to change the `head`.... will fix

Reply:
Addressed by 1d14868